app/views/incomeSources/add/IncomeSourceAddedObligations.scala.html

```diff
@@ -40,6 +42,9 @@
   messages(s"business-added.$key", args: _*)
 }
 
+@getQuarterlyTableHeading(quarterly: Seq[DatesModel]) = @{
+    s"${getMessage("table.heading-1")} ${quarterly.head.inboundCorrespondenceFrom.getYear} ${getMessage("to")} ${quarterly.head.inboundCorrespondenceFrom.getYear + 1}"
```

Comment:
Could fix: you can use parameters within message values by using {0}, this would tidy this block up a bit